playground/schema.js

```diff
@@ -0,0 +1,205 @@
+import _ from 'lodash';
+import minim from 'minim';
+import parseResult from 'minim-parse-result';
+
+const namespace = minim.namespace()
+  .use(parseResult);
+
+// import DataStructureGenerator from 'fury-adapter-swagger/schema';
```

Comment:
NOTE, Following class should be flipped out for an import to fury-adapter-swagger once https://github.com/apiaryio/fury-adapter-swagger/pull/104 is merged and released.